### DISCUSSION TITLE
Alert RegEx Question

### DISCUSSION THREAD

**Discussion by EthanKC** [nextgenhealthcare/connect#6358] [Q&A]
I'm trying to set up alerts on a feed, so they only alert for ACK messages (I got that figured out), but they don't alert for ACKs if the original message was for a particular study description. I figured out the regex for the ACK portion, but the ACK messages we're receiving don't reference the study description at all. Is there any way with alerts to look back at the message that generated the alert, and have the system filter based on that message?

**Comment by jonbartels** [comment]
_A_ way, but maybe not the _best_ way is to route your alerts to a channel. That channel can use the message ID and channel ID from the alert to query for the message for that channel then re-parse the message and pick out additional details.

**Reply by EthanKC** [reply]
Yeah, that might work. Ultimately I ended up telling the client that we couldn't do exactly what he's asking, so instead I'm going to surface the study description in the alert email, and they can filter out what they don't need. Thanks for the response!
